src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -98,6 +130,10 @@ public async Task<bool> PushMetadataAsync(CancellationToken cancellationToken)
                     Log.LogMessage(MessageImportance.High, $"Metadata has been pushed. Build id in the Build Asset Registry is '{recordedBuild.Id}'");
                     Console.WriteLine($"##vso[build.addbuildtag]BAR ID - {recordedBuild.Id}");
 
+                    // Based on the in-memory merged manifest, create a physical XML file and
+                    // upload it to the BlobArtifacts folder
```

Comment:
Why BlobArtifacts instead of the AssetManifests container?

Reply:
Manifest is already been pushed to `AssetManifest` but since now we are treating it as a Blob, we need to push it, also, to the `BlobArtifacts` container

Reply:
I looked at the test build, and I could only find the asset in the BlobArtifacts, and not in the AssetManifests container, so I think it's only being uploaded there?

Reply:
Oh, what I meant was that `Windows_NT-AnyCPU.xml` in the test build is published to AssetManifest. AssetManifets is the base folder we use to generate the MergedManifest so while we could push it to AssetManifests it won't have any effect.

Reply:
I'm just wondering if the right thing to do is to always publish it to assetManifests, and then change the publishing code to always look for a mergedManifest.xml there and upload to blob storage from there, but that's probably not worth it.

Reply:
So I think putting it in blobArtifacts is fine as it conveys more appropriately that it should be published as a blob.

Reply:
I think either approach works. This is probably fine.